### ISSUE TITLE
Preload images option

### ISSUE THREAD

**Issue by docmojoman** [cjboco/cj-image-video-previewer#1]
Is there any way to have the image array load on mouseover?

Also, it would be great if the image array could work with urls as well as file paths!

**Comment by cjboco** [comment]
> Is there any way to have the image array load on mouseover?
> 
> Also, it would be great if the image array could work with urls as well as file paths!

Yes, int theory you could, but the issue would be that you would need time for the images to load in order to have a smooth playback. If you have large images, or slow internet connection for URL's then playback could start before images loaded. OR, if you wait for images loaded, then there could be a delay before any action takes place. 

I could update it to load URL's. Might take awhile, as current workload is pretty heavy.

**Comment by cjboco** [comment]
Images preload now. Issue resolved.
